src/transformers/models/mbart/tokenization_mbart.py

```diff
@@ -96,8 +150,10 @@ def __init__(self, *args, tokenizer_file=None, **kwargs):
         super().__init__(*args, tokenizer_file=tokenizer_file, **kwargs)
 
         self.sp_model_size = len(self.sp_model)
+        language_codes = kwargs.get("language_codes", "CC25")
```

Comment:
Not a fan of this to be honest. We are changing an existing tokenizer here in order to make it work with a new tokenizer that will inherit from this tokenizer. I'm fine with having a new Bart Tokenizer, but since it seems to be very different we should not create an inheritance of `MBartTokenizer`. IMO, this goes a bit against the philosophy of the lib.